external/js/cothority/test/net/net.js

```diff
@@ -0,0 +1,45 @@
+const mocha = require("mocha");
```

Comment:
is mocha unused? Or is it the magic that makes the describe keyword? (Sorry, I am a JS idiot.)

Reply:
Indeed I think it's not needed to import it. The Mocha binary called by `npm run test` so I guess it's already "imported" by default.